pkgs/development/tools/jd-diff-patch/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "jd-diff-patch";
+  version = "1.4.0";
+
+  src = fetchFromGitHub {
+    owner  = "josephburnett";
+    repo   = "jd";
+    rev    = "v${version}";
+    sha256 = "sha256-OAy4IS2JZYYPeJITNHZKiYEapzGqqYPDBCLflLYetzg=";
+  };
+
+  # not including web ui
+  excludedPackages = [ "gae" "pack" ];
+
+  vendorSha256 = "sha256-w3mFra7Je+8qIDQMSyUYucoLZ6GtrZmr56O6xmihIYc=";
+
+  doCheck = true;
+
+  meta = with lib; {
+    description = "jd is a commandline utility and Go library for diffing and patching JSON values";
```

```suggestion
    description = "A commandline utility and Go library for diffing and patching JSON values";
```
Descriptions shouldn't start with the package name, and always with a uppercase letter, as by CONTRIBUTING.md:

>  * Be capitalized.
>  * Not start with the package name.
>  * Not have a period at the end.